### PR TITLE
[zh-cn] sync services-networking/service.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/services-networking/service.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -629,283 +629,129 @@ For example, the names `123-abc` and `web` are valid, but `123_abc` and `-web` a
...
+  the same as if you had requested a Service of `type: ClusterIP`.
 -->
-当你具有需要访问服务的 Pod 时，并且你正在使用环境变量方法将端口和集群 IP 发布到客户端
-Pod 时，必须在客户端 Pod 出现 **之前** 创建服务。
-否则，这些客户端 Pod 将不会设定其环境变量。
-
-如果仅使用 DNS 查找服务的集群 IP，则无需担心此设定问题。
-{{< /note >}}
+`ClusterIP`
+：通过集群的内部 IP 暴露服务，选择该值时服务只能够在集群内部访问。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/service.md:663]:
```suggestion
: 通过集群的内部 IP 暴露服务，选择该值时服务只能够在集群内部访问。
```

**Review by windsonsea — COMMENTED** [review]
Please leave those `:` as is, which is similar with `-` or `*` in an unordered list:

- here
- there
- where

See [preview](https://deploy-preview-41369--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/services-networking/service/) here.

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+Kubernetes also supports DNS SRV (Service) records for named ports.  If the
+`my-service.my-ns` Service has a port named `http` with the protocol set to
+`TCP`, you can do a DNS SRV query for `_http._tcp.my-service.my-ns` to discover
+the port number for `http`, as well as the IP address.
+
+The Kubernetes DNS server is the only way to access `ExternalName` Services.
+You can find more information about `ExternalName` resolution in
+[DNS for Services and Pods](/docs/concepts/services-networking/dns-pod-service/).
+-->
+Kubernetes 还支持命名端口的 DNS SRV（服务）记录。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:1935]:
```suggestion
Kubernetes 还支持命名端口的 DNS SRV（Service）记录。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+如果 `my-service.my-ns` 服务具有名为 `http`　的端口，且协议设置为 TCP，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:1936]:
```suggestion
如果 Service `my-service.my-ns` 具有名为 `http`　的端口，且协议设置为 TCP，
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+则可以对 `_http._tcp.my-service.my-ns` 执行 DNS SRV 查询以发现该端口号、`"http"` 以及 IP 地址。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:1937]:
```suggestion
则可以用 `_http._tcp.my-service.my-ns` 执行 DNS SRV 查询以发现 `http` 的端口号以及 IP 地址。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 5]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+
+You can set the `.spec.internalTrafficPolicy` and `.spec.externalTrafficPolicy` fields
+to control how Kubernetes routes traffic to healthy (“ready”) backends.
+
+See [Traffic Policies](/docs/reference/networking/virtual-ips/#traffic-policies) for more details.
+-->
+### 流量策略
+
+你可以设置 `.spec.internalTrafficPolicy` 和 `.spec.externalTrafficPolicy`
+字段来控制 Kubernetes 如何将流量路由到健康（“就绪”）后端。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:1974]:
```suggestion
字段来控制 Kubernetes 如何将流量路由到健康（“就绪”）的后端。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 6]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+
+<!--
+## Session stickiness
+
+If you want to make sure that connections from a particular client are passed to
+the same Pod each time, you can configure session affinity based on the client's
+IP address. Read [session affinity](/docs/reference/networking/virtual-ips/#session-affinity)
+to learn more.
+-->
+## 粘性会话   {#session-stickiness}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:1986]:
```suggestion
## 会话的黏性   {#session-stickiness}
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 7]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+
+如果你想确保来自特定客户端的连接每次都传递到同一个 Pod，你可以配置根据客户端 IP 地址来执行的会话亲和性。
+阅读[会话亲和性](/zh-cn/docs/reference/networking/virtual-ips/#session-affinity)了解更多。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:1989]:
```suggestion
深入了解可阅读[会话亲和性](/zh-cn/docs/reference/networking/virtual-ips/#session-affinity)。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 8]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+In the example below, the Service named `"my-service"` can be accessed by clients using TCP,
+on `"198.51.100.32:80"` (calculated from `.spec.externalIP` and `.spec.port`).
 -->
 ### 外部 IP  {#external-ips}
 
-如果外部的 IP 路由到集群中一个或多个 Node 上，Kubernetes Service 会被暴露给这些 `externalIPs`。
-通过外部 IP（作为目的 IP 地址）进入到集群，打到 Service 的端口上的流量，
-将会被路由到 Service 的 Endpoint 上。
-`externalIPs` 不会被 Kubernetes 管理，它属于集群管理员的职责范畴。
+如果有路由到一个或多个集群节点的外部 IP，则 Kubernetes 服务可以暴露在这些 `externalIPs` 上。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:2007]:
```suggestion
如果有外部 IP 能够路由到一个或多个集群节点上，则 Kubernetes 服务可以暴露在这些 `externalIPs` 上。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 9]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+当网络流量到达集群时，外部 IP（作为目的 IP 地址）和与该服务匹配的端口、Kubernetes
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:2008]:
```suggestion
当网络流量到达集群时，如果外部 IP（作为目的 IP 地址）和端口都与该 Service 匹配，Kubernetes
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 10]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+配置的规则和路由确保流量被路由到该服务的端点之一。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:2009]:
```suggestion
配置的规则和路由会确保流量被路由到该 Service 的端点之一。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 11]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+
+定义服务时，你可以为任何[服务类型](#publishing-services-service-types) 指定 `externalIPs`。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:2011]:
```suggestion
定义 Service 时，你可以为任何[服务类型](#publishing-services-service-types)指定 `externalIPs`。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 12]
```diff
@@ -1887,37 +1748,271 @@ TLS servers will not be able to provide a certificate matching the hostname that
...
+在下面的例子中，`my-service` 可以在 "`198.51.100.32:80`" (`externalIP:port`) 上被客户端访问。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:2012]: 删除此行

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**